src/Symfony/Component/Console/Input/ArgvInput.php

```diff
@@ -119,9 +119,10 @@ private function parseShortOption($token)
     private function parseShortOptionSet($name)
     {
         $len = \strlen($name);
+        $encoding = \mb_detect_encoding($name);
         for ($i = 0; $i < $len; ++$i) {
             if (!$this->definition->hasShortcut($name[$i])) {
-                throw new RuntimeException(sprintf('The "-%s" option does not exist.', $name[$i]));
+                throw new RuntimeException(sprintf('The "-%s" option does not exist.', \mb_substr($name, $i, 1, $encoding)));
```

Comment:
we put the `\` only for compiler optimized functions (full list available here https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.12/src/Fixer/FunctionNotation/NativeFunctionInvocationFixer.php#L350)

Reply:
Oh, I thought it works for all global functions. Thanks!